app/components/README.md

```diff
@@ -0,0 +1,7 @@
+# Components
+
+This folder contains a collection of components implemented using the [ViewComponent gem](https://viewcomponent.org/). Components are reusable user interface elements, and usually comprise of a component class and ERB template. They are similar to partials in many of their use-cases, but have a few advantages in the form of class-based presenter logic, testability, performance, and conveniences for loading JavaScript assets.
+
+Each component must implement a class extending the `BaseComponent` class. If an accompanying `.html.erb` file exists, it will be used as the template for that component. Similarly, if an `.ts` or `.tsx` file exists, the bundle generated from that JavaScript will be loaded automatically any time the component is rendered.
```

Comment:
do you think it would help to have like a little list or ASCII diagram of which files go where? these are made up names and I'm not even sure if they're the right structure

```
- app/components/MY_COMPONENT/MY_COMPONENT.ts
- app/components/MY_COMPONENT/MY_COMPONENT.html.erb
```

Reply:
> do you think it would help to have like a little list or ASCII diagram of which files go where? these are made up names and I'm not even sure if they're the right structure
> 
> ```
> - app/components/MY_COMPONENT/MY_COMPONENT.ts
> - app/components/MY_COMPONENT/MY_COMPONENT.html.erb
> ```

Yeah, I like the idea of visualizing it a bit, since it's pretty text-heavy as-is.

Reply:
> do you think it would help to have like a little list or ASCII diagram of which files go where? these are made up names and I'm not even sure if they're the right structure

Added in da0c88787.